torch/_dynamo/utils.py

```diff
@@ -1013,30 +1013,29 @@ def record_compilation_metrics(
     exc_type: Optional[Type[BaseException]],
     exc_value: Optional[BaseException],
 ):
-    def us_to_s(field):
-        metric = metrics.get(field, None)
+    def us_to_s(m: Dict[str, Any], field: str) -> Optional[float]:
```

Comment:
Not necessarily needed, but I changed this to pass in the dict instead of assuming the `metrics` parameter of the outer function. Now that we have more than one dict this function operates on, it just seemed like a good way to make a future mistake.